be/src/agent/task_worker_pool.h

```diff
@@ -146,7 +146,7 @@ class TaskWorkerPool {
     static FrontendServiceClientCache _master_service_client_cache;
     static std::atomic_ulong _s_report_version;
 
-    static std::mutex _s_task_signatures_lock;
+    static std::mutex _s_task_signatures_lock[TTaskType::type::TASK_TYPE_COUNT];
```

Comment:
This only cause create more unused object, but I am not find an easy way to solve this problem.
I'm more afraid that this will be ignored when a new type is added later.